# bioRxiv Papers Found on 2025-09-04 (By Category)

Showing top 10 results from the last **2** day(s).
Categories searched: `bioengineering`, `bioinformatics`, `genomics`, `synthetic biology`, `systems biology`, `molecular biology`, `cancer biology`.

---

## [Bioengineering] Graph Autoencoder and StrNN based Causal Analysis of Mortality in Heart Failure Patients

**Posted on:** 2025-09-03

**Abstract:**

> Though analyzed for decades, dissecting and finding mechanisms of cardiovascular diseases, especially heart failures, are still an on-going task for many researchers. However, through recent floods of machine learning and deep learning algorithms to replace traditional approaches, and their applications in diverse cardiovascular research areas, it seems plausible to say that conquering or preventing heart failure catastrophes might no longer be a delusional task within a few more years. To accelerate the arrival of a new era, this research implemented several cutting-edge algorithms currently introduced in causal deep learning to observational heart disease patient data to find key mechanisms that lead to cardiac deaths under a highly flexible framework. Extracting latent causal DAGs from observational data using Graph Auto Encoder, and finding specific causal relationships and interventional effects under Structured Neural Networks (StrNN), novel findings regarding key causes of deaths in heart failure patients were found in numerous aspects. Specifically, existence of intervals where average treatment effects due to causal interventions in platelets, ejection fraction, and serum creatinine levels dramatically decrease or increase was found among heart patients, which can lead to significant eliminations or additions of practical clinical treatments in terms of reducing cardiac death event probability after cardiac failure.

- **Abstract Page:** [https://www.biorxiv.org/content/10.1101/2024.11.11.622921v3](https://www.biorxiv.org/content/10.1101/2024.11.11.622921v3)
- **Direct PDF Link:** [https://www.biorxiv.org/content/10.1101/2024.11.11.622921v3.full.pdf](https://www.biorxiv.org/content/10.1101/2024.11.11.622921v3.full.pdf)

---

## [Bioinformatics] Revising the ortholog conjecture in cross-species comparison of scRNA-seq data

**Posted on:** 2025-09-03

**Abstract:**

> The cross-species comparison of expression profiles uncovers functional similarities and differences between cell types and helps refining their evolutionary relationships. Current analysis strategies typically follow the ortholog conjecture, which posits that the expression of orthologous genes is most similar between species. However, the extent to which this holds true at different evolutionary distances is unknown. Here, we systematically explore the ortholog conjecture in comparative scRNA-seq data. We devise a robust analytical framework, GeneSpectra, to classify genes by expression specificity and distribution across cell types. Our analysis reveals that genes expressed ubiquitously across nearly all cell types exhibit strong conservation of this pattern across species, as do genes with high expression specificity. In contrast, genes within intermediate specificity fluctuate between classes. As expected, ortholog expression becomes more divergent with increased species distance. We also find an overall correlation between similarity in expression profiles and sequence conservation. Finally, our results allow identifying gene classes with highest probability of expression pattern conservation that are most useful for cell type alignment between species. Calibrating reliance on the ortholog conjecture for individual genes, we thus provide a comprehensive framework for the comparative analysis of single cell data.

- **Abstract Page:** [https://www.biorxiv.org/content/10.1101/2024.06.21.600109v3](https://www.biorxiv.org/content/10.1101/2024.06.21.600109v3)
- **Direct PDF Link:** [https://www.biorxiv.org/content/10.1101/2024.06.21.600109v3.full.pdf](https://www.biorxiv.org/content/10.1101/2024.06.21.600109v3.full.pdf)

---

## [Genomics] The human IG heavy chain constant gene locus is enriched for large structural variants and coding polymorphisms that vary among human populations

**Posted on:** 2025-09-03

**Abstract:**

> The human immunoglobulin heavy chain constant (IGHC) domain of antibodies (Ab) is responsible for effector functions critical to immunity. This domain is encoded by genes in the IGHC locus, where descriptions of genomic diversity remain incomplete. We utilized long-read sequencing to build an IGHC haplotype/variant catalog from 105 individuals of diverse ancestry. We discovered uncharacterized single nucleotide variants (SNV) and large structural variants (SVs, n=7), representing new genes and alleles enriched for non-synonymous substitutions, highlighting potential functional effects. Of the 221 identified IGHC alleles, 192 were novel. SNV, SV, and gene allele/genotype frequencies revealed population differentiation, including (i) hundreds of SNVs in African and East Asian populations exceeding a fixation index (FST) of 0.3, and (ii) an IGHG4 haplotype carrying coding variants uniquely enriched in Asian populations. Our results illuminate missing signatures of IGHC diversity and establish a new foundation for investigating IGHC germline variation in Ab function and disease.

- **Abstract Page:** [https://www.biorxiv.org/content/10.1101/2025.02.12.634878v3](https://www.biorxiv.org/content/10.1101/2025.02.12.634878v3)
- **Direct PDF Link:** [https://www.biorxiv.org/content/10.1101/2025.02.12.634878v3.full.pdf](https://www.biorxiv.org/content/10.1101/2025.02.12.634878v3.full.pdf)

---

## [Systems biology] Contextualizing Models: Deriving a Kinetic Model of Cancer Metabolism including Growth via Stoichiometric Reduction

**Posted on:** 2025-09-03

**Abstract:**

> Genome-scale metabolic models (GEMs) offer unprecedented possibilities to study human metabolism, including alterations in cancers. Yet, analyses of GEMs still entail several disadvantages. In particular, constraint-based methods, such as flux balance analysis, are typically restricted to analyse steady-state flux distributions. In contrast, kinetic models based on ordinary differential equations allow assessment of regulatory properties and dynamics. Building kinetic models, however, is still hampered by the lack of knowledge about kinetic parameters and is typically focused on individual pathways. Here, we present an approach to derive kinetic models of metabolism augmented by coarse-grained overall reactions that represent the remaining cellular metabolism and biosynthetic processes. Using algorithmic network reduction, we derive coarse-grained reactions that preserve the correct stoichiometry of precursors, energy, and redox equivalents required for cellular growth. Analysis of the GEM-embedded kinetic model uses Monte Carlo sampling to address parameter uncertainty. We exemplify our approach by constructing a kinetic model of cancer metabolism that includes an explicit description of cellular growth. We show that the GEM-embedded kinetic model differs in its control properties from the corresponding model without growth, with implications for understanding regulatory hotspots and drug target identification.

- **Abstract Page:** [https://www.biorxiv.org/content/10.1101/2025.08.29.672875v1](https://www.biorxiv.org/content/10.1101/2025.08.29.672875v1)
- **Direct PDF Link:** [https://www.biorxiv.org/content/10.1101/2025.08.29.672875v1.full.pdf](https://www.biorxiv.org/content/10.1101/2025.08.29.672875v1.full.pdf)

---

## [Molecular biology] Evaluation of nanopore sequencing for increasing accessibility of eDNA studies in biodiverse countries

**Posted on:** 2025-09-03

**Abstract:**

> Biodiversity loss is a global challenge of the 21st century. Environmental DNA (eDNA)-based metabarcoding offers a cost- and time-efficient alternative to conventional biodiversity surveys, enabling detection of rare, cryptic, and elusive species from environmental samples. However, limited access to genomic technologies restricts the application of eDNA metabarcoding in highly biodiverse remote regions and low- and middle-income countries (LMICs). Here, we directly compared the latest portable nanopore sequencing methods with established Illumina sequencing for vertebrate eDNA metabarcoding of Zambian water samples. Our results show that due to recent improvements in sequencing chemistry and optimized basecalling, nanopore sequencing data can recapitulate or even surpass established protocols, demonstrating the feasibility of in situ biodiversity assessments. eDNA and camera trap-based species detections had minimal overlap in species detections, suggesting a complementary rather than substituting application of these biodiversity monitoring technologies. We finally demonstrate that our entire eDNA workflow can be successfully implemented in a mobile laboratory under remote field conditions by completing all steps--from sample collection to data analysis--within the Luambe National Park in Zambia. This approach has important implications for capacity building in LMICs and for overcoming limitations associated with sample export.

- **Abstract Page:** [https://www.biorxiv.org/content/10.1101/2025.04.21.649756v3](https://www.biorxiv.org/content/10.1101/2025.04.21.649756v3)
- **Direct PDF Link:** [https://www.biorxiv.org/content/10.1101/2025.04.21.649756v3.full.pdf](https://www.biorxiv.org/content/10.1101/2025.04.21.649756v3.full.pdf)

---

## [Cancer biology] A double-negative prostate cancer subtype is vulnerable to SWI/SNF-targeting degrader molecules

**Posted on:** 2025-09-03

**Abstract:**

> Proteolysis targeting chimera (PROTAC) therapies degrading SWI/SNF ATPases offer a novel approach to interfere with androgen receptor (AR) signaling in AR-dependent castration-resistant prostate cancer (CRPC-AR). To explore the utility of SWI/SNF therapy beyond AR-sensitive CRPC, we investigated SWI/SNF-targeting agents in AR-negative CRPC. SWI/SNF targeting PROTAC treatment of cell lines and organoid models reduced the viability of not only CRPC-AR but also WNT-signaling dependent AR-negative CRPC (CRPC-WNT). The CRPC-WNT subgroup represents 11% of around 400,000 cases of CRPC worldwide who die yearly of CRPC. We discovered that SWI/SNF ATPase SMARCA4 depletion interfered with the master transcriptional regulator TCF7L2 (TCF4) in CRPC-WNT. Functionally, TCF7L2 maintains proliferation via the MAPK signaling axis in this subtype of CRPC. These data suggest a mechanistic rationale for interventions that perturb the DNA binding of the pro-proliferative TCF7L2 transcription factor (TF) and/or direct MAPK signaling inhibition in the CRPC-WNT subclass of advanced prostate cancer.

- **Abstract Page:** [https://www.biorxiv.org/content/10.1101/2024.03.24.586276v2](https://www.biorxiv.org/content/10.1101/2024.03.24.586276v2)
- **Direct PDF Link:** [https://www.biorxiv.org/content/10.1101/2024.03.24.586276v2.full.pdf](https://www.biorxiv.org/content/10.1101/2024.03.24.586276v2.full.pdf)

---

## [Bioinformatics] The Demodifier: a tool for screening modification-induced alternate peptide taxonomy in palaeoproteomics

**Posted on:** 2025-09-03

**Abstract:**

> In palaeoproteomic research, the accuracy of taxonomic matches is crucial, as research questions frequently hinge on which species were utilised by ancient people. However, protein modifications including deamidation of glutamine and asparagine, and conversion of glutamine or glutamic acid to pyroglutamic acid, can change the sequence of peptides, leading to spurious taxonomic detections and potentially inaccurate archaeological interpretations. While a handful of examples of this phenomenon have been reported in the literature, the issue is potentially much wider reaching than currently realised. In reality, any time a peptide containing a deamidated glutamine or asparagine residue, an unmodified glutamic acid or aspartic acid residue, or a pyroglutamic acid modification is detected by proteomic search software, the sequence, and therefore potentially its taxonomy, may be incorrect, which could potentially lead to unsound archaeological interpretations. The Demodifier is a fast, open source tool which solves this issue by screening for modification-induced alternate peptide taxonomy, enabling archaeologists to make informed interpretations of the taxonomies of peptides detected in ancient samples. To assess its utility, the Demodifier is tested against an archaeological dataset containing all unique peptides reported in palaeoproteomic studies of dental calculus and vessels. The results reveal that modification-induced alternate peptide taxonomies are severely under-reported, occurring 15% more frequently than previously understood. Modifications were found to produce three different types of inaccurate taxonomic matches: those which yielded completely different taxonomic lowest common ancestors to the input peptide, those which were more taxonomically specific than the input peptide, and those which were less taxonomically specific than the input peptide. The Demodifier therefore enables the rapid detection of potentially inaccurate peptide taxonomies, avoiding spurious archaeological interpretations in future studies.

- **Abstract Page:** [https://www.biorxiv.org/content/10.1101/2025.01.09.632126v4](https://www.biorxiv.org/content/10.1101/2025.01.09.632126v4)
- **Direct PDF Link:** [https://www.biorxiv.org/content/10.1101/2025.01.09.632126v4.full.pdf](https://www.biorxiv.org/content/10.1101/2025.01.09.632126v4.full.pdf)

---

## [Cancer biology] Mice carrying nonsense mutant p53 develop frequent multicentric or metastatic tumors

**Posted on:** 2025-09-03

**Abstract:**

> The TP53 tumor suppressor gene is mutated in a large fraction of human tumors. Close to 11% of TP53 mutations are nonsense mutations, causing premature termination of protein synthesis and expression of truncated inactive p53 protein. The most common TP53 nonsense mutation in human cancer is R213X. To study the impact of TP53 nonsense mutations in vivo, we generated mice harboring the Trp53 nonsense mutation R210X that corresponds to human TP53-R213X. Initially, Trp53R210X mice appear phenotypically normal, although the proportion of female Trp53R210X/R210X mice is dramatically reduced. Female homozygous mice are poor breeders and remain smaller and lighter than female heterozygous and wildtype littermates. Trp53R210X/R210X mice start to show tumors at 2.5 months of age, and their maximal lifespan is 8.5 months. Trp53R210X/+ mice present tumors from 9 months of age, and by 16.5 months of age 50% of all heterozygous mice have developed overt tumors. 71% of tumors from Trp53R210X/+ mice show loss of heterozygosity (LOH). Homozygous mice develop hematopoietic and mesenchymal tumors, most commonly T-cell lymphoma and leiomyosarcoma, and heterozygous mice develop hematopoietic, mesenchymal, epithelial and sex cord tumors, most commonly osteosarcoma and leiomyosarcoma. The tumor phenotype is similar to that of Trp53-null and Trp53-missense knock-in mice, although the Trp53R210X/R210X mice have a high rate of multicentric or metastatic tumors, and Trp53R210X/+ mice have a longer overall survival than Trp53R172H/+ missense mutant knock-in mice. Treatment of T-cell lymphoma cells from Trp53R210X/R210X mice with aminoglycoside G418 induces expression of full-length functional p53 and apoptotic cell death. Our new unique mouse model will allow further studies of the effects of Trp53 nonsense mutation in a multi-organ system and serve as a model for the Li-Fraumeni syndrome (LFS). It will also be valuable for preclinical evaluation of novel therapeutic strategies for targeting TP53 nonsense mutations in cancer.

- **Abstract Page:** [https://www.biorxiv.org/content/10.1101/2025.03.12.642796v2](https://www.biorxiv.org/content/10.1101/2025.03.12.642796v2)
- **Direct PDF Link:** [https://www.biorxiv.org/content/10.1101/2025.03.12.642796v2.full.pdf](https://www.biorxiv.org/content/10.1101/2025.03.12.642796v2.full.pdf)

---

## [Bioinformatics] PG-SCUnK: measuring pangenome graph representativeness using Single-Copy and Universal K-mers

**Posted on:** 2025-09-03

**Abstract:**

> background: Pangenome graphs integrate multiple assemblies to represent non-redundant genetic diversity. However, current evaluations of pangenome graphs rely primarily on technical parameters (e.g., total length, number of nodes/edges, growth curves), which fail to assess how effectively the graph represents homologous stretches across the integrated assemblies. Results: We introduce a novel method to quantitatively assess how well a pangenome graph represents its integrated assemblies. Our method quantifies how many single-copy and universal k-mers from the source assemblies are uniquely and completely represented within the graph nodes. Implemented in the open-source tool PG-SCUnK, this approach identifies the fractions of unique, duplicated, and split k-mers, which correlate with short read mapping rates to the pangenome graph. Conclusions: Insights provided by PG-SCUnK facilitate the selection of appropriate parameters to build optimal pangenome graphs. Availability and implementation: A bash implementation of the PG-SCUnK workflow is freely available under the GNU GPLv3 license at https://github.com/cumtr/PG-SCUnK/.

- **Abstract Page:** [https://www.biorxiv.org/content/10.1101/2025.04.03.646777v3](https://www.biorxiv.org/content/10.1101/2025.04.03.646777v3)
- **Direct PDF Link:** [https://www.biorxiv.org/content/10.1101/2025.04.03.646777v3.full.pdf](https://www.biorxiv.org/content/10.1101/2025.04.03.646777v3.full.pdf)

---

## [Genomics] Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma.

**Posted on:** 2025-09-03

**Abstract:**

> Ewing sarcoma (EwS) is a fusion-driven malignancy, peaking in adolescence. Although EwS tumors are driven uniquely by EWS::FLI1 and related fusions, patient outcomes vary greatly. If and how tumor plasticity of EWS::FLI1-regulated transcriptional signatures contribute to disease progression is not known. To address this, we utilized a single-cell co-assay of RNA and chromatin accessibility (ATAC) sequencing to identify gene regulatory networks in EwS. By comprehensively characterizing regulatory elements across cell lines, we identified multiple unique modules of gene regulation. Differential usage and prevalence of these modules was evident across cell lines, associated with distinct epigenetic and transcriptomic signatures, and in specific cases, modifiable by exogenous TGF-B;. When we examined primary EwS patient tumors, we observed these same regulatory modules were variably enriched both across and within tumors, highlighting the existence of intratumoral heterogeneity in gene regulatory networks. Our findings demonstrate that multiple, co-existing transcriptional programs shape the phenotypic diversity of EwS and suggest that the balance between these networks may have important implications for clinical outcomes and targeted therapy development.

- **Abstract Page:** [https://www.biorxiv.org/content/10.1101/2025.06.18.660457v2](https://www.biorxiv.org/content/10.1101/2025.06.18.660457v2)
- **Direct PDF Link:** [https://www.biorxiv.org/content/10.1101/2025.06.18.660457v2.full.pdf](https://www.biorxiv.org/content/10.1101/2025.06.18.660457v2.full.pdf)

